xonsh/tools.py

```diff
@@ -633,17 +617,17 @@ def bool_seq_to_csv(x):
 
 
 def is_completions_display_value(x):
-    return x in {'none', 'single', 'multi'}
+    return x in ('none', 'single', 'multi')
```

Comment:
though [here](https://wiki.python.org/moin/PythonSpeed#Use_the_best_algorithms_and_fastest_tools) it says that sets are faster for membership lookups. maybe that rather apllies to large data.